### DISCUSSION TITLE
ImportError from ObjectContactsView when running the manage.py migrate command

### DISCUSSION THREAD

**Discussion by dumbledong** [mattieserver/netbox-topology-views#328] [General]
**NetBox version**

3.5.1

**Topology Views version**

3.4 - unfortunately I don't know exactly where to check

**Steps to Reproduce**

When running the following command (from /opt/netbox/netbox within virtual env)

`python3 manage.py migrate netbox_topology_views`

**Expected Behavior**

The command should complete without error, then I should be able to run
`python3 manage.py collectstatic --no-input`

**Observed Behavior**

Receive the error
`ImportError: cannot import name 'ObjectContactsView' from 'tenancy.views' (/opt/netbox-3.5.1/netbox/tenancy/views.py)`

![netbox error](https://github.com/mattieserver/netbox-topology-views/assets/8128627/c1bc22a0-3928-4143-99a5-a92299443248)
:

**Comment by mattieserver** [comment]
I see /opt/netbox-3.5.1/ but you run the command from /opt/netbox/netbox, is this a softlink?

**Reply by dumbledong** [reply]
It is.

Ohhhh boy, you know what? The script throwing the exception is in the package directory for netbox-inventory. I'll take my leave.
